.github/workflows/codeql-analysis.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
-  schedule:
-    - cron: '0 4 * * 3'
```

Comment:
Any reason why the cron job was removed?
Maybe add a comment in the PR

Reply:
Added a note.